mlir/test/Dialect/Arith/canonicalize.mlir

```diff
@@ -701,6 +701,15 @@ func.func @extFPConstant() -> f64 {
   return %0 : f64
 }
 
+// CHECK-LABEL: @extFPVectorConstant
+//       CHECK:   %[[cres:.+]] = arith.constant dense<[0.000000e+00, 1.000000e+00]> : vector<2xf128>
+//       CHECK:   return %[[cres]]
+func.func @extFPVectorConstant() -> vector<2xf128> {
```

Comment:
Can we also add the test case where folding does not happen (e.g. nan, negative zero)?

Reply:
This is the case for f8m2e5fnuz and f16 but I don't know how to find/generate the exact number:
```
  EXPECT_FALSE(APFloat::isRepresentableAsNormalIn(APFloat::Float8E5M2FNUZ(), APFloat::IEEEhalf()));
```

Any suggestions on getting the constant? Maybe @krzysz00 knows?

Reply:
@Lewuathe I added a TODO for this since I don't see an easy way to produce the exact value. I'm not worried about the correctness of the conversion function itself because I already have a test for `truncf` that captures this. I hope this addresses your concerns.